src/backend/central/central_crud.py

```diff
@@ -164,18 +195,72 @@ def generate_updated_xform(
     return outfile
 
 
-def create_QRCode(project_id=None, token=None, name=None):
+def create_QRCode(
+    project_id: int,
+    token: str,
+    name: str,
+):
     """Create the QR Code for an app-user"""
     appuser = OdkAppUser()
     return appuser.createQRCode(project_id, token, name)
 
 
-def upload_media(project_id: int, xform_id: str, filespec: str):
+def upload_media(
+    project_id: int,
+    xform_id: str,
+    filespec: str
+):
     """Upload a data file to Central"""
     xform.uploadMedia(project_id, xform_id, filespec)
 
 
-def download_media(project_id: int, xform_id: str, filespec: str):
+def download_media(
+    project_id: int,
+    xform_id: str,
+    filespec: str
+):
     """Upload a data file to Central"""
     filename = "test"
     xform.getMedia(project_id, xform_id, filename)
+
+def convert_csv(
+    filespec: str,
+    data: bytes,
+):
+    """Convert ODK CSV to OSM XML and GeoJson"""
```

Comment:
Awesome, very happy that we're offering GeoJSON right away (not everyone wants to deal with OSM XML).